### ISSUE TITLE
Showing available packs in Item Selection modal can be confusing

### ISSUE THREAD

**Issue by roxy-dao** [msupply-foundation/open-msupply#1227]
## What went wrong? 😲
Showing available number of packs in the item selection modal can be confusing since users might assume that there is only `x` number of stock left in store until they select an item.

In stocktake, the item selection shows available number of packs, but the stocktake line modal shows total number of packs. The numbers can differ at times, and lead to confusion

![image](https://user-images.githubusercontent.com/29744386/222277699-d3897932-38f6-476a-9a4c-d1d16fb101d4.png)
Figure: The last column shows the total quantity of available units rather than the total in store

## Expected behaviour 🤔
Show `in store` packs instead?

**Comment by roxy-dao** [comment]
UI updates don't make this confusing anymore. Will close and reopen if it comes up again.

**Comment by marthakeezy** [comment]
Same in Prescriptions ^

**Comment by Chris-Petty** [comment]
Isn't the opposite of what @marthakeezy experienced? In prescription you want to know how much stock you have available when trying to select items, but the item DDL list shows SOH (only after adding it do you realise there is no stock available to issue).

**Comment by roxy-dao** [comment]
Moving some telegram comments here: 

Chris: Yea defo makes sense to show available rather than SOH
Craig: I tend to disagree - if its 500 in stock and 0 available, there’s still 500 in the warehouse - until you ship the order it could still be accessed. This is a rare case anyway - and also it’s clear to see what the situation is.
Chris: There's probably a cool way can show available and soh without over crowding it

**Comment by andreievg** [comment]
Refinement, primary show available, but need indication of how much is 'total' (or the difference between available and total when they differ) -> product/design decision

**Comment by louisaw123** [comment]
Agree with @roxy-dao this is very confusing. Firstly, that item search is missing a header to indicate what these values actually represent if it is not representing my expectation.

In terms of expectation, for stocktakes, I would expect to see my SOH rather than what is the available stock (this is really not useful in this situation).

My concern with it right now is that it is misleading and so a user could easily use this search bar as a guide and look at an item that says 0 units and think that _"I don't need to go looking for this and counting it because there is none in stock"_. But they have not even selected the item line to see the snapshot quantity. This will then lead to errors with stocktake and counts being missed. 

Not only that, they will also be confused with the 2 different values from the search bar and the snapshot quantity and not know the meanings of these values and which one to trust , especially if their count does not match the snapshot quantity.

In conclusion, suggest that the units in that search match the snapshot units

**Comment by roxy-dao** [comment]
This issue should encompass all areas where the item search is used~. Used stocktake as an example above but the same component is used everywhere (except for Requisition and Internal Order which use a similiar but different component)
